### PR TITLE
entry on yecao by Lu Xun done

### PR DESCRIPTION
**Pull request by LenaHenningsen**
I hope there are no conflicts with yesterdays pull request. Will continue tonight.

### CODE REVIEW

File: data/ActType.csv [hunk 1]
```diff
@@ -1,6 +1,9 @@
 action_ID,action_name,created,created_by,last_modified,last_modified_by
 AT01,to read,2017-07-12,OS,2017-07-12,OS
-AT02,to copy,2017-07-12,OS,2017-07-12,OS
+AT02,to copy (shouchao),2017-07-12,OS,2017-07-28,LH
```
Comment (duncdrum) [inline data/ActType.csv]: I'd suggest ``to transcribe`` to stick with english ActTypes

Comment (LenaHenningsen) [inline data/ActType.csv]: Yes, I thought of that too, but shouchao so neatly captures more than copy. And as I am writing a book on shouchao practices, I'd insist even though it is not for purists.

Comment (duncdrum) [inline data/ActType.csv]: I actually like the idea of having a typography of shuochao practices in here. Still think we shouldn't cope out so easily by using pinyin. if there are different ways of transcribing you want to clearly distinguish we need to find terms for them. Probably better to open an issue for the discussion though.

Comment (LenaHenningsen) [inline data/ActType.csv]: Thanks. I think I'll merge the stuff now myself otherwise I will end up with conflicts if I continue now.

**Pull request merged**